### PR TITLE
restore tmate workflow

### LINKED ISSUE #56: Restore tmate.yml for debugging.
Restore [`tmate.yml`](https://github.com/bluesky/hklpy/blob/e572c19820325921d8142dc7a3bfbe407f07421c/.github/tmate.yml#L1) for use when debugging on Github Actions, but leave it _inactive_ by placing it in the `<root>/.github/` directory.

Move the YAML file into the `workflows` subdirectory and push to start a tmate session.

_Originally posted by @prjemian in https://github.com/bluesky/hklpy/issues/54#issuecomment-738399460_

### PR DESCRIPTION
**Pull request by prjemian**
fixes #56

Not active until you want it.  Copy into workflows directory to activate.

### CODE REVIEW

File: .github/tmate.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: CI
+
+# Copy into the workflows subdirectory to use this script.
+# see: https://github.com/marketplace/actions/debugging-with-tmate
```
Comment (mrakitin) [inline .github/tmate.yml:4]: 👍 thanks for the instructions on how to use it.

File: .github/tmate.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        # python-version: [3.6, 3.7, 3.8]
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: set environment name in YAML file
+      shell: bash -l {0}
+      run: |
+        sed -i.bak "s/name: ${PACKAGE}/name: ${ENV_NAME}/g" environment.yml
```
Comment (mrakitin) [inline .github/tmate.yml:27]: What does `-i.bak` mean?

**Review by mrakitin — APPROVED** [review]
Looks good to me!

File: .github/tmate.yml (hunk 2 above)
Comment (prjemian) [reply .github/tmate.yml:27]: Makes a local backup copy of the original file.  Useful as a diagnostic (with tmate, for example).

Comment (mrakitin) [reply .github/tmate.yml:27]: Thanks! I did not know about that option, pretty useful.

Comment (prjemian) [reply .github/tmate.yml:27]: StackOverflow to the rescue, once again!

**Comment by prjemian** [comment]
Thanks for the quick review!

**Pull request merged**